.github/workflows/lint.yml

```
@@ -41,14 +41,4 @@ jobs:
         run: docker compose run api yarn lint-es
 
       - name: Running Prettier
-        run: docker compose run api yarn lint-prettier
-
-      - name: Generating OpenAPI definition
-        run: docker compose -f docker-compose.yml -f docker-compose.apigen.yml run apigen && docker compose down
-
-      - name: Generating Redoc
-        uses: peaceiris/actions-gh-pages@v3
-        with:
-          personal_token: ${{ secrets.DACOCOBOT_PAT }}
-          publish_dir: ./openapi
-          publish_branch: docs
+        run: docker compose run api yarn lint-prettier
```

[low-prio non-blocking nit-pick]
```suggestion
        run: docker compose run api yarn lint-prettier

```